src/main/java/org/assertj/core/api/FloatingPointNumberAssert.java

```diff
@@ -150,4 +152,148 @@
    * @throws AssertionError if the actual value is equal to {@code NaN}.
    */
   SELF isNotNaN();
+
+  /**
+   * Verifies that the actual value is a finite floating-point value.
+   * <p>
+   * Example:
+   * <blockquote><pre>{@code
+   * // assertions will pass
+   * assertThat(+1.0f).isFinite();
+   * assertThat(-1.0d).isFinite();
+   *
+   * // assertions will fail
+   * assertThat(Float.NaN).isFinite();
+   * assertThat(Float.NEGATIVE_INFINITY).isFinite();
+   * assertThat(Float.POSITIVE_INFINITY).isFinite();
+   * assertThat(Double.NaN).isFinite();
+   * assertThat(Double.NEGATIVE_INFINITY).isFinite();
+   * assertThat(Double.POSITIVE_INFINITY).isFinite();
+   * }</pre></blockquote>
+   *
+   * @return this assertion object.
+   * @throws AssertionError if the actual value is not a finite floating-point value.
+   * @see #isInfinite()
+   */
+  SELF isFinite();
+
+  /**
+   * Verifies that the actual value represents positive infinity or negative infinity.
+   * <p>
+   * Example:
+   * <blockquote><pre>{@code
+   * // assertions will pass
```

Comment:
replace all `<blockquote><pre>{@code` by `<pre><code class='java'>` and join lines with one space after `<code class='java'>` so for this one it should look like:
```
<pre><code class='java'> // assertions will pass
```